core/xml.js

```diff
@@ -445,8 +445,9 @@ const domToWorkspace = function(xml, workspace) {
   }
 
   // Disable workspace resizes as an optimization.
-  if (workspace.setResizesEnabled) {
-    workspace.setResizesEnabled(false);
+  // Assume it is rendered so we can check.
+  if (/** @type {!WorkspaceSvg} */ (workspace).setResizesEnabled) {
+    /** @type {!WorkspaceSvg} */ (workspace).setResizesEnabled(false);
```

Comment:
WorkspaceSvg [requires XML](https://github.com/google/blockly/blob/20f1475afc1abf4b5e600219c2981150fc621ba5/core/workspace_svg.js#L27) [for pasting](https://github.com/google/blockly/blob/20f1475afc1abf4b5e600219c2981150fc621ba5/core/workspace_svg.js#L1574), so this would also cause a circular dependency :/

🎶 everything is a bundle of properties 🎶